src/libraries/System.ObjectModel/src/System/ComponentModel/TypeConverterAttribute.cs

```diff
@@ -30,7 +32,7 @@ public TypeConverterAttribute()
         /// class, using the specified type as the data converter for the object this attribute
         /// is bound to.
         /// </summary>
-        public TypeConverterAttribute(Type type)
+        public TypeConverterAttribute([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] Type type)
```

Comment:
See [my comment here](https://github.com/dotnet/runtime/issues/39125#issuecomment-657816918), this should be `PublicConstructors` instead since the converter can have a ctor that takes the underlying `Type` as well.

Reply:
It might be nice to also test a type that uses a ctor that takes an underlying `Type`, e.g. [NullableConverter](https://github.com/dotnet/runtime/blob/master/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/NullableConverter.cs) or [EnumConverter](https://github.com/dotnet/runtime/blob/master/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs).